code/_globalvars/game_modes.dm

```diff
@@ -9,3 +9,7 @@ GLOBAL_VAR(custom_event_msg)
 GLOBAL_VAR(custom_event_admin_msg)
 
 GLOBAL_LIST_EMPTY(summon_spots)
+
+GLOBAL_VAR(protect_target) // The one sucker that will be fought over by antags
+GLOBAL_LIST_EMPTY(protect_target_assassins) // Who is going to hunt them
+GLOBAL_LIST_EMPTY(protect_target_protectors) // Who is going to protect them
```

Comment:
If this mini-gamemode is suppose to be traitor only, I'm thinking these would be better as vars on 
`datum/gamemode/traitor`. And then can be accessed by `SSticker.mode.protect_target` etc.

Reply:
I thought about putting them on the gamemode but I decided to make them global to make expansion possible for later if we want to get rid of mode bound antags and such.
I'll think about it though since currently finding who is the target can be a bit annoying as admin. Having to navigate the global variables and the like (or just finding somebody with protect)

Reply:
Changed it in the latest commit. It made more sense to put it on the gamemode datum in the end. Also avoids more globals that could be put under other objects